app/models/freak.rb

```diff
@@ -2,6 +2,7 @@
 
 class Freak < ApplicationRecord
   has_one :photo, as: :imageable, dependent: nil
+  has_many :projects, as: :proj, dependent: nil
```

Comment:
Please avoid incomplete terms (e.g. `proj`) as they can be unclear. I don't think the `as` or `dependent` options are needed here either.